bundles/org.openhab.binding.dscalarm/src/main/java/org/openhab/binding/dscalarm/internal/handler/DSCAlarmBaseBridgeHandler.java

```diff
@@ -380,7 +380,7 @@ public void checkThings() {
                 }
 
             } else {
-                logger.error("checkThings(): Thing handler not found!");
+                logger.error("checkThings(): Thing handler not found! Thing: {} {}", thing.getUID(), thing);
```

Comment:
I have not checked what returns thing.toString() but what interesting additional info provides your second parameter ?

I see logger.error is massively used in that class while WARN level should have been used. Will make probably no sense to change only this one.

Reply:
As the cases do not have affect on openHAB as a whole, this should be changed to warn (some others might even to debug)

Reply:
@lolodomo I've removed the additional argument thing.toString().

Reply:
also changed all logger.error to logger.warn